src/main.js

```diff
@@ -0,0 +1,49 @@
+const container = document.querySelector('.container');
+
+function CreateTodosList(url) {
+    let xhr = new XMLHttpRequest();
+    xhr.open('GET', url, false);
```

Comment:
You use sync mode, it's bad. It will not be working in production.
https://learn.javascript.ru/ajax-xmlhttprequest#sinhronnye-i-asinhronnye-zaprosy

Reply:
use async mode or promise